dockers/docker-database/database_config.json

```diff
@@ -0,0 +1,24 @@
+{
+    "DISABLE_DATABASE": {
+        "redis":{
+            "port": 6379,
+            "socket": "/var/run/redis/redis.sock",
+            "databases" : ["CONFIG_DB","STATE_DB","LOGLEVEL_DB","SYSMON_DB"]
```

Comment:
what is SYSMON_DB? I do not think we have defined this one.

Reply:
removed